doc/sources/quick-start.rst

```diff
@@ -14,17 +14,455 @@
+
+Before you begin, make sure that you have read and write permissions for Scikit-learn files. 
+
+With global patching, you can:
+
+.. list-table:: 
+   :header-rows: 1
+   :align: left
+
+   * - Task
+     - Action
+     - Note
+   * - Patch all supported algorithms
+     - Run this command:
+
+       :: 
+         
+          python -m sklearnex.glob patch_sklearn
+     
+     - If you run the global patching command several times with different parameters, then only the last configuration is applied.
+   * - Patch selected algorithms
+     - Use ``--algorithm`` or ``-a`` keys with a list of algorithms to patch. For example, to patch only ``SVC`` and ``RandomForestClassifier`` estimators, run
+
+       ::
+ 
+           python -m sklearnex.glob patch_sklearn -a svc random_forest_classifier
+  
+     -   
+   * - Enable global patching via code
+     - Use the ``patch_sklearn`` function with the ``global_patch`` argument:
+
+       ::
+
+          from sklearnex import patch_sklearn
+          patch_sklearn(global_patch=True)
+          import sklearn
+      
+     - After that, Scikit-learn patches is enabled in the current application and in all others that use the same environment.
+   * - Disable patching notifications
+     - Use ``--no-verbose`` or ``-nv`` keys:
+
+       ::
+
+          python -m sklearnex.glob patch_sklearn -a svc random_forest_classifier -nv
+     -  
+   * - Disable global patching
+     - Run this command:
+
+       ::
+
+          python -m sklearnex.glob unpatch_sklearn
+     -
+   * - Disable global patching via code
+     - Use the ``global_patch`` argument in the ``unpatch_sklearn`` function
+
+       ::
+
+          from sklearnex import unpatch_sklearn
+          unpatch_sklearn(global_patch=True)
+     -
+    
+.. tip:: If you clone an environment with enabled global patching, it will already be applied in the new environment.
+
+Unpatching
+**********************
+
+To undo the patch (also called `unpatching`) is to return scikit-learn to original implementation and
+replace patched algorithms with the stock scikit-learn algorithms.
+
+To unpatch successfully, you must reimport the scikit-learn package::
+
+  sklearnex.unpatch_sklearn()
+  # Re-import scikit-learn algorithms after the unpatch
+  from sklearn.cluster import KMeans  
+
+
+Installation 
+--------------------
+
+.. contents:: :local:
+
+.. tip:: To prevent version conflicts, we recommend to create and activate a new environment for |intelex|. 
+
+Install from PyPI 
+**********************
+
+Recommended by default. 
+
+To install |intelex|, run:
+
+::
+
+  pip install scikit-learn-intelex
+
+**Supported Configurations**
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - OS / Python version
+     - Python 3.8
+     - Python 3.9
+     - Python 3.10
+     - Python 3.11
+     - Python 3.12
+   * - Linux* OS
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
+     - [CPU, GPU]
```

Comment:
Question for the team - are we supporting GPU for 3.8/3.11/3.12 at this point if up-to-date `dpctl` is not available?